index.rst

```diff
@@ -0,0 +1,19 @@
+.. _acrn_home:
+
+OPEA Project Documentation
+##########################
+
+Welcome to the OPEA Project (|version|) documentation published |today|.
+OPEA streamlines implementation of enterprise-grade Generative AI by efficiently
+integrating secure, performant, and cost-effective Generative AI workflows into business value.
```

Comment:
s/into/to deliver

Reply:
I grabbed that text from https://opea.dev/ :)